doc/cascadia/SettingsSchema.md

```diff
@@ -137,7 +137,8 @@ Bindings listed below are per the implementation in `src/cascadia/TerminalApp/Ap
 - moveFocusRight
 - moveFocusUp
 - moveFocusDown
-- toggleFullscreen
+- toggleFullscreen
+- Find
```

Comment:
```suggestion
- find
```
and propagate these changes to the serialization. The current styling is in camelcase

Reply:
changed